server/ladders-local.ts

```diff
@@ -292,11 +288,13 @@ class LadderStore {
 
 			let reasons = '' + (Math.round(p1newElo) - Math.round(p1elo)) + ' for ' + (p1score > 0.9 ? 'winning' : (p1score < 0.1 ? 'losing' : 'tying'));
 			if (reasons.charAt(0) !== '-') reasons = '+' + reasons;
-			room.addRaw(Chat.html`${p1name}'s rating: ${Math.round(p1elo)} &rarr; <strong>${Math.round(p1newElo)}</strong><br />(${reasons})`);
+			room.addRaw(Chat.html`${p1name}'s rating: ${Math.round(p1elo)} &rarr; ` +
```

Comment:
```suggestion
			room.addRaw(
				Chat.html`${p1name}'s rating: ${Math.round(p1elo)} &rarr; <strong>${Math.round(p1newElo)}</strong><br />(${reasons})`
			);
```

Reply:
That's >120 chars though :|

Reply:
It's not though... tslint counts tabs as one char each. (Also as mentioned elsewhere, we carve out a tslint exception for strings so they are always allowed on one line.)